<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceControlWidget</class>
 <widget class="QWidget" name="DeviceControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="deviceListGroupBox">
     <property name="title">
      <string>Devices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="serialOutputComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="addSerialDevicePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="serialOutputListWidget"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="removeSerialDevicePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="cueSizeGroupBox">
     <property name="title">
      <string>Upload Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Configuration size (in bytes)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="configSizeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="sendPushButton">
        <property name="text">
         <string>Send to Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="interpretCuePushButton">
        <property name="text">
         <string>Show Cuefile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
